eng/tools/rush-runner/src/env.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
only real change here was caching the base dir so we could grab it from multiple modules instead of having to pass it around through args